sdf/1.7/pose.sdf

```diff
@@ -4,7 +4,16 @@
 
   <attribute name="relative_to" type="string" default="" required="*">
     <description>
-      Name of frame relative to which the pose is applied.
+      If specified, this pose is expressed in the named frame. The named frame
+      must be declared both within the same file, and in the same scope
+      (world/model) as the element that has its pose specified by this tag.
+
+      If missing, this pose is expressed in the scope's implicit frame. This
+      frame is implicitly defined by the world or model that contains the
```

Comment:
The implicit frame used depends on the context of the pose. For `//visual` and `//collision`, their pose is relative to the parent `//link`, so saying the implicit frame is defined by the world or model is not accurate.

Reply:
@azeey Thanks, I didn't know that. Do you know if the default frame (by context) is documented somewhere? Perhaps we can link to it, or I can come up with a way of phrasing it consciesly here once I know the cases.

Reply:
Yes, it's documented in http://sdformat.org/tutorials?tut=pose_frame_semantics&ver=1.5&cat=specification&#parent-frames-in-sdf-1-4. The later versions of the SDFormat spec simply keep that behavior when `relative_to` is not specified.

Reply:
I updated the description to 

```
 If missing, the pose is expressed in the frame of the parent element. The
parent element is the grandparent of this pose element, i.e., the parent of
the element that has its pose set by this pose element.
```

I think the latter part is a bit cryptic and sounds artificial; I didn't manage to come up with nicer wording. Suggestions for improvement are welcome :)

Reply:
How about
>If missing, the pose is expressed in the frame of the parent XML element of the element that contains the pose. For exceptions to this rule and more details on the default behavior, see http://sdformat.org/tutorials?tut=pose_frame_semantics&ver=1.5 and http://sdformat.org/tutorials?tut=pose_frame_semantics&ver=1.7

The exceptions are:
* `//joint/pose`: the default pose is relative to the child link/frame
* `//frame/pose`: the default pose is relative to the attached_to frame.

Also, I think just saying parent element can be confusing for joints, which have parents and children, so I changed it to "parent XML element".

Reply:
Sounds very nice; I've updated the description.

I'm actually a bit back and forth on adding links. I really like the cross-referencing of the spec and the docs; when I first got into SDFormat I didn't realize that documentation exists to the extent it does and I wouldn't have known where to look. 

At the same time, the current spec website doesn't allow clicking hyperlinks in descriptions (because the entire element is a hyperlink to the element's fragment, see for example: http://sdformat.org/spec?ver=1.8&elem=sensor#camera_distortion). As a result, I am using the chrome debug tool to extract links from the spec which is sub-optimal. Perhaps something low-priority to look into in the future :)